src/Features/VisualBasic/Portable/CodeRefactorings/InlineTemporary/InlineTemporaryCodeRefactoringProvider.vb

```diff
@@ -74,9 +74,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeRefactorings.InlineTemporary
                 Dim solution = document.Project.Solution
                 Dim findReferencesResult = Await SymbolFinder.FindReferencesAsync(local, solution, cancellationToken).ConfigureAwait(False)
 
-                Dim locations = findReferencesResult.Single(Function(r) Equals(r.Definition, local)).Locations
-                If Not locations.Any(Function(loc) semanticModel.SyntaxTree.OverlapsHiddenPosition(loc.Location.SourceSpan, cancellationToken)) Then
-                    Return locations
+                Dim referencedSymbol = findReferencesResult.SingleOrDefault(Function(r) Equals(r.Definition, local))
+                If referencedSymbol Is Nothing Then
+                    Return SpecializedCollections.EmptyEnumerable(Of ReferenceLocation)()
```

Comment:
this seems like a bug.  how can we not have an entry for the very item we're doing FindReferences on?

Reply:
@CyrusNajmabadi FYI, the C# version have the exact same null check.
Probably checking the C# file history in GitHub and seeing when the check was added would provide more information?

Reply:
The C# check was introduced in https://github.com/dotnet/roslyn/commit/31fc1118a2a25c8e365674d9f334c6f4291d9bce#diff-009129e38bd68fdf74e5a3b09866f329

Reply:
@allisonchou Pinging you as you're the author of this commit.

https://github.com/dotnet/roslyn/commit/31fc1118a2a25c8e365674d9f334c6f4291d9bce#diff-009129e38bd68fdf74e5a3b09866f329

Were you able to know why/when the result from FindReferencesAsync is empty?

Reply:
No.  We haven't been able to determine this.  It shoudl not happen.  I put through a PR to try to find some potential causes, but we've gotten no hits off of it yet.

Reply:
@CyrusNajmabadi I hit this again while debugging.

![image](https://user-images.githubusercontent.com/31348972/95082915-10f67c80-071c-11eb-861f-4cf973845b23.png)

Reply:
@CyrusNajmabadi I hit this again while debugging.

![image](https://user-images.githubusercontent.com/31348972/95082915-10f67c80-071c-11eb-861f-4cf973845b23.png)

Reply:
@CyrusNajmabadi Note that it reproduces **constantly** for me at the same line.

Reply:
Can you please file a repro then :-)

Reply:
Unfortunately, couldn't reproduce again ;-(
It was reproducing constantly within the same debugging session.